CONTRIBUTING.md

```diff
@@ -0,0 +1,269 @@
+<head>
+  <meta charset="UTF-8">
+  <meta name="description" content="Contributing to rocWMMA">
+  <meta name="keywords" content="ROCm, contributing, rocWMMA">
+</head>
+
+# Contributing to rocWMMA #
+
+We welcome contributions to rocWMMA.  Please follow these details to help ensure your contributions will be successfully accepted.
+
+## Issue Discussion ##
+
+Please use the [GitHub Issues](https://github.com/ROCm/rocWMMA/issues) tab to notify us of issues.
+
+* Use your best judgement for issue creation. If your issue is already listed, upvote the issue and
+  comment or post to provide additional details, such as how you reproduced this issue.
+* If you're not sure if your issue is the same, err on the side of caution and file your issue.
+  You can add a comment to include the issue number (and link) for the similar issue. If we evaluate
+  your issue as being the same as the existing issue, we'll close the duplicate.
+* If your issue doesn't exist, use the issue template to file a new issue.
+    * When filing an issue, be sure to provide as much information as possible, including script output so
+      we can collect information about your configuration. This helps reduce the time required to
+      reproduce your issue.
+    * Check your issue regularly, as we may require additional information to successfully reproduce the
+      issue.
+* You may also open an issue to ask questions to the maintainers about whether a proposed change
+  meets the acceptance criteria, or to discuss an idea pertaining to the library.
+
+New issues should use the following templates:
+
+* **Bugs**
+    + Any unintended functionality within the rocWMMA library should be thoroughly documented with the template below.
+        1. Description: ***Please be clear and concise***
+        2. Steps for Reproduction:
+            - Hardware Information:
+            - Docker Environment or Software Versions:
+            - Expected Behavior:
+            - Actual Behavior:
+        3. Any additional information:
+* **Enhancement Requests**
+    + Any proposed enhancements to rocWMMA should be thoroughly documented with the template below.
+        1. Description: ***Please be clear and concise***
+        2. Value and Motivation
+            - Feature and/or Functionalities Enabled:
+            - Any Alternatives
+        3. Any additional information:
+
+## Acceptance Criteria ##
+
+The goal of rocWMMA is to provide a C++ API for accelerating matrix multiply accumulate (MMA) operations utilizing AMD GPU hardware.
```

Comment:
The goal of rocWMMA is to provide a C++ API for facilitating block-wise decomposition of matrix multiply accumulate (MMA) workflows while leveraging specialized AMD GPU hardware. rocWMMA also facilitates migration of nvcuda::wmma users to AMD's HIP environment on AMD GPUs and provide a level of equivalent functionality.